sheetfu/modules/table.py

```diff
@@ -35,6 +36,25 @@ def __iter__(self):
     def __getitem__(self, index):
         return self.items[index]
 
+    @staticmethod
+    def get_table_from_sheet(spreadsheet, sheet_name, notes=False, backgrounds=False, font_colors=False):
+        """
+        Method to create a table from a whole sheet of a spreadsheet.
+        This method assumes the header row is 1.
```

Comment:
I think it would be good to have the header_row argument in this function aswell, in the equivalent function of this in GAS we use that parameter a lot.

Reply:
The main idea was to have a method that handles tablers from whole sheets, but I guess having it couldn't hurt.